test_scripts/Resumption/Handling_errors_from_HMI/047_Another_RPC_IVD_RAI2_before_request_to_HMI.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description:
+-- In case:
+-- 1. App1 is subscribed to <RPC1> and <RPC2>, App2 is subscribed to <RPC2>
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App1 and App2 re-register with actual HashId
+-- 4. SDL starts resumption for App1 and App2:
+--    <RPC1> and <RPC2> requests related to App1 are sent from SDL to HMI
+-- 5. HMI responds with error resultCode to <RPC1> and success to <RPC2>
+-- 6. SDL doesn't send revert <RPC2> request to HMI
+-- 7. SDL doesn't restore subscription to <RPC2> for App1 and responds RAI_Response(success=true,resultCode=RESUME_FAILED)
+-- 8. SDL restores subscription to <RPC2> for App2 and responds RAI_Response(success=true,resultCode=SUCCESS) to App2
```

Comment:
@GetmanetsIrina Does SDL not send <RPC2> request to restore App2?

Reply:
yes, SDL should not send the request to restore App2 because success  resultCode is already received for <RPC2>